TestBenches/ProjectionCalculator_test.cpp

```diff
@@ -59,14 +57,12 @@ int main()
   TrackletProjectionMemory<BARRELPS> tproj_barrel_ps[TC::N_PROJOUT_BARRELPS];
   TrackletProjectionMemory<BARREL2S> tproj_barrel_2s[TC::N_PROJOUT_BARREL2S];
   TrackletProjectionMemory<DISK> tproj_disk[TC::N_PROJOUT_DISK];
+  TrackletParameterMemory trackletParametersOut;
 
   // print the input files loaded
```

Comment:
@aryd Please remove the corresponding comment.
```suggestion
```

Reply:
Done.